src/views/Locations.vue

```diff
@@ -151,14 +188,95 @@ export default defineComponent({
       });
       return popover.present();
     },
+    async getFacilities(vSize?: any, vIndex?: any) {
+      const viewSize = vSize ? vSize : process.env.VUE_APP_VIEW_SIZE;
+      const viewIndex = vIndex ? vIndex : '0';
+
+      const payload = {
+        "inputFields": {
+          "facilityId_value": `%${this.queryString}%`,
+          "facilityId_op": "like",
+          "facilityId_ic": "Y",
+          "facilityId_grp": "1",
+          "facilityName_value": `%${this.queryString}%`,
+          "facilityName_op": "like",
+          "facilityName_ic": "Y",
+          "facilityName_grp": "2",
+        } as any,
+        "fieldList": [],
+        viewSize,
+        viewIndex,
+        "entityName": "ProductStoreFacilityDetail",
+        "noConditionFind": "Y",
+        "distinct": "Y"
+      }
+
+      this.store.dispatch('util/getFacilities', payload);
+    },
+    async fetchFacilityTypes() {
+      let resp;
+      try{
+        const payload = {
+          "inputFields": {
+            "parentTypeId": "VIRTUAL_FACILITY",
+            "parentTypeId_op": "notEqual",
+            "facilityTypeId": "VIRTUAL_FACILITY",
+            "facilityTypeId_op": "notEqual",
+          },
+          "fieldList": ["description", "facilityTypeId"],
+          "viewSize": 100,
+          "entityName": "FacilityType",
+          "noConditionFind": "Y",
+          "distinct": "Y"
+        }
+
+        resp = await UtilService.fetchFacilityTypes(payload);
+        if(resp.status === 200 && resp.data.docs?.length && resp.data.docs?.length > 0 && !hasError(resp)) {
+          this.facilityTypes = this.facilityTypes.concat(resp.data.docs);
+        }
+      } catch(error) {
+        console.error(error);
+        showToast(translate("Something went wrong"));
+      }
+    },
+    async fetchEcomStores() {
+      let resp;
+
+      try{
+        const payload = {
+          "viewSize": 100,
+          "fieldList": ["productStoreId", "storeName"],
+          "entityName": "ProductStore",
+          "distinct": "Y",
+          "noConditionFind": "Y"
+        }
+
+        resp = await UtilService.fetchEcomStores(payload);
+        if(resp.status === 200 && resp.data.docs?.length && resp.data.docs?.length > 0 && !hasError(resp)) {
+          this.eComStores = this.eComStores.concat(resp.data.docs);
```

Comment:
Why we are concatenating?

Reply:
Sir, as I have discussed with @dt2patel Sir that we need to show All initially in both filters, if we want to use ion-select and want to show the initial value in UI we need to add the All option in the filters list that's why I have added All in both filter's prop data instead of adding it in response which we are getting from API is this a right approach WDYT please suggest me @adityasharma7 Sir.